galerautils/src/gu_byteswap.h

```diff
@@ -58,6 +58,11 @@ static GU_FORCE_INLINE uint64_t GU_ROTL64 (uint64_t x, int8_t r)
 #  define gu_bswap16 __bswap16_var
 #  define gu_bswap32 __bswap32_var
 #  define gu_bswap64 __bswap64_var
+#elif defined(__OpenBSD__)
+#include <endian.h>
+#define gu_bswap16(x) __swap16(x)
+#define gu_bswap32(x) __swap32(x)
+#define gu_bswap64(x) __swap64(x)
 #elif defined(__sun__)
```

Comment:
Robert used the swapX macros. Wonder if that makes more sense as they use the bswapX macros further down.